### PR TITLE
Include function arguments in drop function command to fix (#1548), fix permission tab (#1549)

### LINKED ISSUE #1548: issue with drop function in postgres < 10
Drop function feature fails for postgres versions < 10.

`drop function` needs full definition in versions < 10 .

 See https://www.postgresql.org/docs/10/sql-dropfunction.html vs https://www.postgresql.org/docs/9.6/sql-dropfunction.html)

### LINKED ISSUE #1549: issue with custom function permission tab
If SETOF table is in another schema, the schema of the table is not considered will generating the permissions page link

### PR DESCRIPTION
**Pull request by karthikvt26**
fix #1548, Fix #1549 
### Description

What component does this PR affect? 

- [ ] Server
- [x] Console
- [ ] CLI
- [ ] Docs
- [ ] Community Content
- [ ] Build System

Requires changes from other components? If yes, please mark the components:

- [ ] Server
- [ ] Console
- [ ] CLI
- [ ] Docs
- [ ] Community Content
- [ ] Build System

### Type

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs update
- [ ] Community content

### Checklist:

- [x] I have read the **[contributing guide](https://github.com/hasura/graphql-engine/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
- [ ] This change requires a change in the documentation. 
- [ ] I have updated the documentation accordingly.
- [ ] I have added required tests.

### CODE REVIEW

**Comment by karthikvt26** [comment]
Requires changes in the server code which will be fixed in #1547

File: console/src/components/Services/Data/Function/Permission/Permission.js [hunk 1]
```diff
@@ -78,14 +82,14 @@ class Permission extends React.Component {
           applicable to the data returned by this function
         </p>
         <div className={styles.commonBtn}>
-          <Link to={permissionTableUrl}>
+          <a href={permissionTableUrl}>
```
Comment (praveenweb) [inline console/src/components/Services/Data/Function/Permission/Permission.js]: This needs to be later changed back to <Link>.

Comment (karthikvt26) [reply console/src/components/Services/Data/Function/Permission/Permission.js]: Link breaks this functionality due to a data requirement in permissions tab being not satisfied. I will fix this issue along with the permission data issue in a different PR

Comment (praveenweb) [reply console/src/components/Services/Data/Function/Permission/Permission.js]: Created an issue for the same #1567 and approving this PR.

**Pull request merged**